css-flexbox-1/flex-shrink-008.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Flexbox Test: flex-shrink - less than one</title>
+<link rel="author" title="Geoffrey Sneddon" href="mailto:me@gsnedders.com">
+<link rel="help" title="7.3.1. The 'flex-shrink' property" href="http://www.w3.org/TR/css-flexbox-1/#propdef-flex-shrink">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="">
+<meta name="assert" content="Test checks that remaining free space is calculated from 'flex-shrink' set to positive number less than one.">
+<style>
+  .container {
+    background-color: red;
+    height: 50px;
+    width: 100px;
+  }
+  #test1, #test2 {
+    display: flex;
+    height: 50px;
+    width: 50px;
+  }
+  #test1 > div, #test2 > div {
+    background-color: green;
+    height: 50px;
+  }
+  #test1 > div {
+    flex-shrink: 0.9;
+    width: 550px;
+  }
+  #test2 > div {
+    flex-shrink: 0.25;
+    width: 75px;
```

Comment:
I've done the maths for this three times and got three different results. (I've gone wrong in different places each time!) From the one time I got what every browser does, can someone else confirm that this doesn't run into (undefined) rounding issues?